examples/simple-auth/src/components/PrivateRoute.js

```diff
@@ -1,12 +1,12 @@
 import React from "react"
 import PropTypes from "prop-types"
-import { navigate } from "@reach/router"
+import { redirectTo } from "@reach/router"
```

Comment:
Would you be able to try [this approach](https://www.gatsbyjs.org/docs/gatsby-link/#programmatic-navigation)?

That's our recommended approach! (Not sure why it wasn't using that previously 😅 )

Reply:
Yes, of course, I'll do that as soon as I'm off the 🚋

Reply:
@DSchau should I change the other reference to `redirectTo` as well? (In the Login component?)

Reply:
@Siyfion yes please :)

Reply:
Yup, all working fine.